gunicorn/config.py

```diff
@@ -605,15 +605,17 @@ class WorkerClass(Setting):
 
         The default class (``sync``) should handle most "normal" types of
         workloads. You'll want to read :doc:`design` for information on when
-        you might want to choose one of the other worker classes.
+        you might want to choose one of the other worker classes. Required
+        libraries may be installed using setuptools' extra_require feature.
 
         A string referring to one of the following bundled classes:
 
         * ``sync``
-        * ``eventlet`` - Requires eventlet >= 0.9.7
-        * ``gevent``   - Requires gevent >= 0.13
-        * ``tornado``  - Requires tornado >= 0.2
+        * ``eventlet`` - Requires eventlet >= 0.9.7 (extra gunicorn[eventlet])
```

Comment:
Maybe something like "(or install it via \`\`gunicorn[...]\`\`)" or "(or install it via \`\`pip install gunicorn[...]\`\`)" would make it clearer.

Reply:
Thanks for the suggestion, I wasn't happy with my wording myself.